files/cf-exclude-include.json

```diff
@@ -4,24 +4,29 @@
     "auudio-forge",
     "appleskin",
     "armor-toughness-bar",
+    "beehive-tooltips",
     "better-advancements",
     "betterf3",
+    "bettertags",
     "better-third-person",
     "biomeinfo",
     "block-drops-jei-addon",
     "cherished-worlds",
     "clickable-advancements",
+    "configured",
     "controlling",
     "ctm",
     "custom-main-menu",
     "defensive-measures",
     "ding",
     "drippy-loading-screen",
     "dynamiclights-reforged",
+    "effective_fg",
     "embeddium",
     "embeddium-extension",
     "embeddium-extras",
     "enchantment-descriptions",
+    "entity_collision_fps_fix",

```

Comment:
For future reference, these slugs need to match exactly as they are in CurseForge system

![image](https://github.com/itzg/docker-minecraft-server/assets/988985/86e87923-4402-44d2-a0b2-be1293b48cf7)